src/colors.st.css

```diff
@@ -1,82 +1,86 @@
 :vars {
 
   /* DARK - Primary */
-  D10: #162d3d; /* Primary Text */
+  D10: #162D3D; /* Primary Text */
   D20: #32536A; /* Secondary Text */
   D30: #577083;
-  D40: #7a92a5; /* Placeholder Text */
-  D50: #b6c1cd; /* Disabled - Text + System Icons */
-  D55: #cbd3dc; /* Disabled button - new version */
-  D60: #dfe5eb; /* Dividers, Disabled - Borders */
-  D70: #f0f4f7; /* Page Background, Disabled - Background */
-  D80: #ffffff; /* Content box background */
+  D40: #7A92A5; /* Placeholder Text */
+  D50: #B6C1CD; /* Disabled - Text + System Icons */
+  D55: #CBD3DC; /* Disabled button - new version */
+  D60: #DFE5EB; /* Dividers, Disabled - Borders */
+  D70: #F0F4F7; /* Page Background, Disabled - Background */
+  D80: #FFFFFF; /* Content box background */
 
   /* BLUE - Brand */
-  B00: #2b81cb; /* Button Focus */
-  B10: #3899ec; /* Main, Button */
-  B20: #4eb7f5; /* Hover for elements with B10, buttons, Notifications */
+  B00: #2B81CB; /* Button Focus */
+  B05: #308DDD; /* CTA */
```

Comment:
why B05?
Where did it came from?

Reply:
It's in the zeplin spec:
https://app.zeplin.io/project/5864e02695b5754a69f56150/screen/58ac1742739bb1a59277bd7e